client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -147,8 +148,12 @@ const UpgradeNudge = React.createClass( {
 } );
 
 export default connect(
-	state => ( {
-		site: getSelectedSite( state ),
-	} ),
+	( state, ownProps ) => {
+		const siteId = getSelectedSiteId( state );
```

Comment:
Theoretically the selector should be able to handle falsey args. We also don't render in the case when site doesn't exist. https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrade-nudge/index.jsx#L72